assets/src/components/Lightbox/Lightbox.js

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import PhotoSwipe from '../../../../node_modules/photoswipe/dist/photoswipe.js';
+import PhotoSwipeUI_Default from '../../../../node_modules/photoswipe/dist/photoswipe-ui-default.js';
+
+// `items` should be an array of object with this shape:
+// [{ src, w, h, title }, ...]
+// See: https://photoswipe.com/documentation/getting-started.html
+export const Lightbox = ({ index, isOpen, items, onClose }) => {
+  let photoSwipeElement = useRef(null);
+
+  const photoSwipeOptions = {
+    index: index || 0,
+    closeOnScroll: false,
+    history: false
+  };
+
+  const [currentIndex, setCurrentIndex] = useState(photoSwipeOptions.index);
+
+  if (!isOpen) {
+    onClose();
```

Comment:
I don't think we need to call this here. `onClose` was already called [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/414/files#diff-7603fc2cbf7cc1eec26221206dffa1f05c92e5b18163c0ab259a2b83443d1767R44) and/or [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/414/files#diff-7603fc2cbf7cc1eec26221206dffa1f05c92e5b18163c0ab259a2b83443d1767R48) at this point, right?

Reply:
Hmm actually `onClose` just sets `isOpen` to false :smiley: https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/a21971d3f325a396b5c360a342d5a234183d4fa2/assets/src/components/Lightbox/useLightbox.js#L13